app/presenters/content_workflow_presenter.rb

```diff
@@ -0,0 +1,35 @@
+class ContentWorkflowPresenter < CSVPresenter
+  def initialize(scope = Edition.not_in(state: ["archived"]).all)
+    super(scope)
+    self.column_headings = [
+      :content_title,
+      :content_slug,
+      :content_url,
+      :current_status,
+      :stage,
+      :format,
+      :current_assignee,
+      :created_at,
+    ]
+  end
+
+private
+
+  def build_csv(csv)
+    csv << column_headings.collect { |ch| ch.to_s.humanize }
+    scope.each do |item|
```

Comment:
@tuzz At the moment we are only searching for published content.. not too sure about the size.. however, it would be useful to batch things up.